ape_safe/accounts.py

```diff
@@ -678,6 +686,16 @@ def submit_safe_tx(
         Returns:
             ``ReceiptAPI``
         """
+        if not isinstance(safe_tx, (SafeTxV1, SafeTxV2)):
+            safe_tx_id = safe_tx
+            safe_tx = self.client.get_safe_tx(safe_tx_id).as_safe_tx(
+                version=str(self.version), chain_id=self.chain_manager.chain_id
```

Comment:
We may want to allow `chain_id` param in case we are not connected in Ape we can still submit a safe tx